src/components/announcementsList/index.tsx

```diff
@@ -1,65 +1,155 @@
-import { Row } from 'antd';
-import React, { useEffect, useState } from 'react';
+import { range } from 'lodash';
+import { default as React, useState } from 'react';
```

Comment:
Woah woah, `import { default as React`? Is this intentional?

Reply:
ugh its just how my VSCode extension does it, I dont really like it either but I dont think it matters _too_ much?? The more I look at it the more annoyed I get though 🤮